### PR TITLE
Make XX:IdleTuningGcOnIdle part of Xtune:virtualized

### LINKED ISSUE #124: Add GC tuning on Idle part of Xtune:virtualized
JVM idle tuning/optimization is not included as part of Xtune:virtualized.  It needs to enabled/set as another separate option(XX:+IdleTuningGcOnIdle). As this option will be beneficial to cloud/virtualized environment specific workloads, it is good to include into Xtune:virtualized optimizations.

### PR DESCRIPTION
**Pull request by Param-S**
Make IdleTuningGcOnIdle optimization is covered/included part of the
option(Xtune:virtualized) which grouped the optimizations relevant for virtualized
environment.

closes #124
Signed-off-by: Parameswaran Selvam <parselva@in.ibm.com>

### CODE REVIEW

File: runtime/vm/jvminit.c [hunk 1]
```diff
@@ -1823,7 +1823,8 @@ IDATA VMInitStages(J9JavaVM *vm, IDATA stage, void* reserved) {
 				IDATA argIndexIgnoreUnrecognizedOptionsEnable = FIND_AND_CONSUME_ARG(EXACT_MATCH, VMOPT_XXIDLETUNINGIGNOREUNRECOGNIZEDOPTIONSENABLE, NULL);
 				IDATA argIndexIgnoreUnrecognizedOptionsDisable = FIND_AND_CONSUME_ARG(EXACT_MATCH, VMOPT_XXIDLETUNINGIGNOREUNRECOGNIZEDOPTIONSDISABLE, NULL);
 
-				if (argIndexGcOnIdleEnable > argIndexGcOnIdleDisable) {
+				/* idle heap tuning is enabled only if -XX:+IdleTuningGcOnIdle or -Xtune:virtualized is set as VM options*/
```
Comment (DanHeidinga) [inline runtime/vm/jvminit.c]: This should be limited to Java 9 only.  We don't want to change the default for this on Java 8.

File: runtime/vm/jvminit.c [hunk 2]
```diff
@@ -1823,7 +1823,8 @@ IDATA VMInitStages(J9JavaVM *vm, IDATA stage, void* reserved) {
...
+				if ((argIndexGcOnIdleEnable > argIndexGcOnIdleDisable) || ((argIndexGcOnIdleDisable == -1) && J9_ARE_ANY_BITS_SET(vm->runtimeFlags, J9_RUNTIME_TUNE_VIRTUALIZED))) {
```
Comment (DanHeidinga) [inline runtime/vm/jvminit.c]: For readability, we typically format this larger if check like:
```
if ((argIndexGcOnIdleEnable > argIndexGcOnIdleDisable)
|| ((argIndexGcOnIdleDisable == -1) && J9_ARE_ANY_BITS_SET(vm->runtimeFlags, J9_RUNTIME_TUNE_VIRTUALIZED))
) {
```

But the check will likely change when the Java 9 only check is added.

**Comment by Param-S** [comment]
@DanHeidinga Have scoped the change "inclusion of IdleTuningGcOnIdle into Xtune:virtualized" to Java 9 & above & updated "if condition check" as per coding standard.

**Comment by DanHeidinga** [comment]
Jenkins test sanity

**Pull request merged**